data/binding/mapbinding.go

```diff
@@ -1,17 +1,23 @@
 package binding
 
 import (
+	"errors"
 	"reflect"
 
 	"fyne.io/fyne"
 )
 
+var (
+	errKeyNotFound = errors.New("key not found")
+	errParseFailed = errors.New("format did not match 1 value")
```

Comment:
nit: this error isn't used by this file, consider moving to convert.go

Reply:
I was trying to keep useful consts like this out of generated files. In this case there was no equivalent non-generated convert file. I guess `binding.go` is another option.

Reply:
Yeah maybe we should put all the errors together in the package-named file